### DISCUSSION TITLE
How to find the source code of the ecole.core?

### DISCUSSION THREAD

**Discussion by dashijia** [ds4dm/ecole#323] [Questions]
I want to see the source code of the ecole components, such as ecole.instance, ecole.scip and so on.
Where should I find them ?

**Comment by AntoinePrv** [comment]
Hi @dashijia,

The source code of `ecole.core` is in `python/ecole/src/ecole/core` (the pathname is a bit verbose as there is quite a bit going on).
You'll notice it's only C++ files, because this are the Python bindings of the C++ library `libecole`.
You won't find any implementations here, you'll just see how the Python functions map to the C++ `libecole` ones.
